### PR TITLE
add unicorn drop + misc

### PR DESCRIPTION
**Pull request by zelinarxy**
- fix a typo in "remilia"
- tweak the project badge style to allow for multiple rows
- add the unicorn airdrop (missing a lot of information, not sure how much of it is available but i'll get in contact with the team)

### CODE REVIEW

File: src/components/cards/AirdropCard.svelte [hunk 1]
```diff
@@ -104,7 +104,8 @@
 						</div>
 					</aside>
 				{/if}
-				{#if thisDrop.claim.dates.claimDateClose >= new Date() - 7 * 24 * 60 * 60 * 1000}
+				{#if thisDrop.claim.dates.claimDateClose === null}
```
Comment (zelinarxy) [inline src/components/cards/AirdropCard.svelte]: i'm not sure what the end date is, but without tweaking the template here it said the claim was closed

File: src/components/cards/AirdropCard.svelte [hunk 2]
```diff
@@ -208,7 +209,7 @@
 					class="grid grid-cols-{thisDrop.claim.eligibility.derivsIncluded.length >= 4
 						? 3
 						: thisDrop.claim.eligibility.derivsIncluded
-								.length} w-full gap-x-5 text-[9px] md:text-[11px]"
+								.length} w-full gap-x-5 gap-y-2 text-[9px] md:text-[11px]"
```
Comment (zelinarxy) [inline src/components/cards/AirdropCard.svelte:219]: there was no gap between rows previously

File: src/components/cards/AirdropCard.svelte [hunk 3]
```diff
@@ -183,22 +184,22 @@
...
 				{/if}
+				<div class="flex items-center justify-center font-bold">Airdrop end</div>
+				<div class="flex items-center justify-center">
+					{thisDrop.claim.dates.claimDateClose ? new Date(thisDrop.claim.dates.claimDateClose).toLocaleString(undefined, {
+						day: '2-digit',
+						month: '2-digit',
+						year: 'numeric',
+						hour: '2-digit',
+						minute: '2-digit'
+					}) : "unknown"}~
```
Comment (zelinarxy) [inline src/components/cards/AirdropCard.svelte]: kind of went in two directions here: hard coded `unknown` as a fallback for `claimDateClose`, but left `claimDuration` as-is and put `unknown` as the value in the config

lmk if you prefer one or the other or a secret third thing

File: src/airdrops/2-UNICORN.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      "usdValueClaimOpen": null
+    },
+    "eligibility": {
+      "isAllocationScaledPerNft": null,
+      "derivsIncluded": [
+        "Yayo",
+        "Radbro",
+        "Milady",
+        "RedactedRemilioBabies",
+        "PixeladyMaker",
```
Comment (zelinarxy) [inline src/airdrops/2-UNICORN.json]: formatte this, without the spaces, cause that was the vibe as far as i could tell? idk some of them look weird

File: src/airdrops/2-UNICORN.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    "image": "https://unicorn.meme/img/uni-head.webp"
+  },
+  "claim": {
+    "dates": {
+      "claimDateOpen": "2024-04-02T00:00:00.000Z",
+      "claimDateClose": null,
+      "claimDuration": "unknown"
+    },
+    "value": {
+      "airdroppedTokens": null,
```
Comment (zelinarxy) [inline src/airdrops/2-UNICORN.json]: i don't know any of this. `null` violates the typing in `src/lib/types.ts` but nothing appears to have broken? lmk how you want me to handle

**Comment by zelinarxy** [comment]
btw i don't have access to see the vercel preview, so if that's atrociously broken i'm blissfully ignorant

File: src/components/cards/AirdropCard.svelte (hunk 1 above)
Comment (Elliott-Green) [reply src/components/cards/AirdropCard.svelte]: Nice!

File: src/components/cards/AirdropCard.svelte (hunk 2 above)
Comment (Elliott-Green) [reply src/components/cards/AirdropCard.svelte:219]: Nice!

File: src/components/cards/AirdropCard.svelte (hunk 3 above)
Comment (Elliott-Green) [reply src/components/cards/AirdropCard.svelte]: I made this more explict in template logic. Bit more code, less to digest looking at.

File: src/airdrops/2-UNICORN.json (hunk 4 above)
Comment (Elliott-Green) [reply src/airdrops/2-UNICORN.json]: Nice, I reordered them in a more hierarchial order in my head and updated template.

File: src/airdrops/2-UNICORN.json (hunk 5 above)
Comment (Elliott-Green) [reply src/airdrops/2-UNICORN.json]: I made the types attributes more explict with the null type and can be safe to check against that in the template.

**Comment by Elliott-Green** [comment]
> btw i don't have access to see the vercel preview, so if that's atrociously broken i'm blissfully ignorant

running build+preview should be a good enough simulation as to what vercel will run :)

**Pull request merged**